velox/exec/RowContainer.cpp

```diff
@@ -471,7 +467,7 @@ void RowContainer::freeRowsExtraMemory(
 }
 
 void RowContainer::invalidateColumnStats() {
-  if (!collectColumnStats_ || rowColumnsStats_.empty()) {
+  if (rowColumnsStats_.empty()) {
```

Comment:
If this is enabled by default, maybe we can consider to repalce `columnHasNulls_` with this also.

Reply:
Yeah that is true but this light weighted column stats will be voided when erasure happens.